Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemComponent.cpp

```diff
@@ -424,6 +424,49 @@ namespace AzToolsFramework
             return newTemplateId;
         }
 
+        void PrefabSystemComponent::UpdateTemplateFilePath(TemplateId templateId, const AZ::IO::PathView& filePath)
+        {
+            auto findTemplateResult = FindTemplate(templateId);
+            if (!findTemplateResult.has_value())
+            {
+                AZ_Warning(
+                    "Prefab", false,
+                    "PrefabSystemComponent::UpdateTemplateFilePath - "
+                    "Template associated by given Id '%llu' doesn't exist in PrefabSystemComponent.",
+                    templateId);
+                return;
+            }
+
+            if (!filePath.IsRelative())
+            {
+                AZ_Warning(
```

Comment:
This should be an AZ_Error since the action won't be completed if a relative path is not provided. Same with the other warning in this function.